### ISSUE TITLE
Pedtask is not being detected

### ISSUE THREAD

**Issue by Vampiiree** [multitheftauto/mtasa-blue#2406]
**Which resource(s) have this problem?**
I don't think it's a specific resource.

**Describe the bug**
Basically, this is a glitch where it allows the player to run slightly faster than normal and walk infinitely without holding any keys.

**To Reproduce**
1. Punch once so it triggers the TASK_SIMPLE_FIGHT
2. Hold W to walk while on the "fighting mode"
3. Use /jetpack (preferably through a bind)
4. See if the ped starts to move faster
5. Remove the jetpack

**Expected behavior**
If you can reproduce what was mentioned above, you will notice that you start walking quickly and even faster if you start running. 

**Screenshots**
https://www.youtube.com/watch?v=bT-XhcDveb8

**Version**
v1.5.9-release-21005

**Additional context**
I used MTA wiki's code to check which pedtasks were getting trigged because I don't know anything about coding, but I think blocking TASK_SIMPLE_JETPACK while performing TASK_SIMPLE_FIGHT should fix this?

**Comment by bum8hj** [comment]
Similar to [#1470](https://github.com/multitheftauto/mtasa-blue/issues/1470)

Oh, and this should've been posted on [mtasa-blue](https://github.com/multitheftauto/mtasa-blue/issues)

**Comment by Vampiiree** [comment]
Yes, it could be the bug, but his video isn't available so I can't be sure about that as he didn't mention the fact that the player runs faster. Also at 0:47 you can notice that equipping the jetpack again doesn't remove the animation.

Sorry about posting it in the wrong place. It's my first time here :anguished:

**Comment by lopezloo** [comment]
Please report this in multitheftauto/mtasa-blue project.

**Comment by patrikjuvonen** [comment]
I've transferred the issue to mtasa-blue.
